app/src/main/java/com/cornellappdev/score/components/UpcomingGameCard.kt

```diff
@@ -137,7 +120,14 @@ fun UpcomingGameCard(
             sportIcon = sportIcon,
             topCornerRound = false,
             modifier = modifier
```

Comment:
Okay wait this is a little sus. You're giving the `modifier` argument to this `SportCard` AND the containing `Column`. That feels wrong; I'm sure you meant to give this one just a new `Modifier` and then containing one the argument `modifier`.